### PR TITLE
feat: card always flip animate in collection view

### PR DESCRIPTION
**Pull request by tomski80**
card always rotate in collection view now, there is no change how it behave in lesson view.

### CODE REVIEW

File: src/components/Card.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-<div class="flip-container pointer" @click="flip" :class="{ flip: !newCard, flipper: !newCard, flipperFast: newCard }">
+<div class="flip-container pointer" @click="flip" :class="flipClass">
```
Comment (AntonTymczenko) [inline src/components/Card.vue:2]: I like this reduction! Neat!

File: src/components/Card.vue [hunk 2]
```diff
@@ -27,13 +27,27 @@ export default {
     },
     cardSide () {
       return this.question ? this.card.q : this.card.a
+    },
+    flipClass () {
+      if(this.endlessFlip){
```
Comment (AntonTymczenko) [inline src/components/Card.vue]: I suggest refactoring this part. See my later commit

**Review by AntonTymczenko — APPROVED** [review]

File: src/components/Card.vue [hunk 3]
```diff
@@ -27,15 +26,19 @@ export default {
     },
     cardSide () {
       return this.question ? this.card.q : this.card.a
+    },
+    flipClass () {
+      return {
```
Comment (AntonTymczenko) [inline src/components/Card.vue:31]: I've refactored this part a little bit, so it is more straightforward and written in less code

**Review by AntonTymczenko — APPROVED** [review]

**Review by svmi3195 — APPROVED** [review]
I couldn't test it at work, but the code looks good.

**Comment by AntonTymczenko** [comment]
Ok, let's wait for Tom's comment. If it's all OK – let's merge

**Comment by tomski80** [comment]
I can't test it right now, but from the code your changes looks good. If it works correctly for you then it is good to go.

**Pull request merged**